@kiva/kv-components/vue/KvLocationSelector.vue

```diff
@@ -68,7 +68,7 @@
 		</fieldset>
 		<div class="tw-hidden md:tw-flex tw-gap-2 tw-justify-end tw-mt-1.5">
 			<button
-				class="tw-text-link"
+				class="tw-text-black"
```

Comment:
Does not having a class result in the same expected text color?

Reply:
oh yeah, nice call, ty!